daemon/src/taker_cfd.rs

```diff
@@ -60,6 +60,21 @@ enum RollOverState {
     None,
 }
 
+pub struct RollOverParams {
+    settlement_interval: Duration,
+    /// The time since cfd creation where rollover is triggered
+    rollover_interval: Duration,
```

Comment:
To me, the docs and variable name here still don't make sense to me.

As far as I understand, this interval defines the minimum time that has to pass for a CFD to be automatically rolled over. How about this:

```suggestion
    /// Defines the minimum duration a CFD needs to be open for until a roll-over is triggered.
    min_open_until_rollover: Duration,
```

cc @da-kami @bonomat

Reply:
I find it confusing if I have to think about the expiry from the beginning. 
I would rather think about it from the end, i.e. the CFD should always have a remaining expiry of 24h, if it doesn't, roll over.

Reply:
>  I would rather think about it from the end, i.e. the CFD should always have a remaining expiry of 24h, if it doesn't, roll over.

That is not how it is currently implemented though. I based my suggestion on how things are implemented. If we implement it differently, I am happy to go with a name such as `min_time_to_expiry`.

Reply:
Do we even need this? Shouldn't this always be the same value as `Order#settlement_time_interval_hours`?

Reply:
How would you know when to  trigger the rollover?